### PR TITLE
removing permissions for statuses

### PR DESCRIPTION
**Pull request by Kamahl19**
it looks like we have not used that permission in any file. if you find it please let me know

### CODE REVIEW

File: root/socialnet/install_sn.php [hunk 1]
```diff
@@ -656,14 +656,20 @@
...
 			'phpbbSN_create_fms_primarygroups',
 		),
-
-		'cache_purge'		 => array(
+	),
+	
+	'0.7.1'	 => array(
+	
+		'permission_remove'	 => array(
+			array('u_sn_userstatus', true),
```
Comment (senky) [inline root/socialnet/install_sn.php:664]: Use tabs, please. Everywhere.

Comment (Kamahl19) [inline root/socialnet/install_sn.php:664]: I am using tabs

File: root/socialnet/install_sn.php [hunk 2]
```diff
@@ -656,14 +656,20 @@
...
+		),
+	
+  	'cache_purge'		 => array(
```
Comment (senky) [inline root/socialnet/install_sn.php:667]: this line uses spaces - 2 spaces and 1 tab. It needs to be rewritten to use 2 tabs.

File: root/socialnet/install_sn.php (hunk 1 above)
Comment (senky) [inline root/socialnet/install_sn.php:664]: Ok sorry, github displays it in strange way...

**Comment by Culprit-CZ** [comment]
Why remove we permissions? What is the BUG?
**Permissions are used.**
When permission for module exist, than is used.

Better will be add permissions for all other modules

> u_sn_im
> u_sn_notify

I think that for FMS module, should be allways set permission to ALLOW.

**Comment by senky** [comment]
In fact you are true. It maybe should not be removed, but for sure it needs to be set to "Yes" by default.

**Comment by Kamahl19** [comment]
So what is your conclusion? Do you both want permissions or not? If yes, how can I discard changes from this PR ?

**Comment by senky** [comment]
I think we should keep it. If it is not used in core of SN, this does not mean add-ons will not want to use it.

**Comment by Culprit-CZ** [comment]
I'm for **discard** and **add permissions** for another modules and set it for group registered usesrs by default enabled

**Comment by Kamahl19** [comment]
but what about other groups? it is useless and admins have to set it after installing. then they are asking why statuses are not working...

**Comment by senky** [comment]
This issue is not about that. We should deal with permissions for statuses only. They should be set to "Yes" by default. That is it.

**Comment by Kamahl19** [comment]
All right so how can I discard changes I made to remove it ?

**Comment by senky** [comment]
Simply replace them by reverting changes. Or delete commits, it should revert the changes made in the commit.

**Pull request closed without merging**